.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -3,37 +3,52 @@ Soft code freeze for a release is the *second Wednesday of the sprint*, and most
+    - [ ] If strings have been provided by content team, open a PR with the newly provided strings. Update their `strings:needed` labels to `approved:strings`
+    - [ ] Otherwise, escalate to Product because this may miss the release.
 
-### Product Integrity [Thursday, 2nd week of sprint]
+## Soft code freeze [Wednesday, 2nd week of sprint]
+- [ ] Merge any remaining [automated L10N string PRs](https://github.com/mozilla-mobile/fenix/pull/6156).
+- [ ] Create a branch off of master (DO NOT PUSH YET) for the *current* milestone of format `releases/v2.3` (where 2.3 is the *current* milestone). After that, anything landing in master will be part of the next milestone.
+- [ ] On the Release branch, pin the AC version to the stable version ([example](https://github.com/mozilla-mobile/fenix/commit/e413da29f6a7a7d4a765817a9cd5687abbf27619)) with commit message "Issue #`<this releng issue>`: Pin to stable AC `<version>` for release v2.3" (replacing 2.3 with the version)
+- [ ] [Create an issue](https://github.com/mozilla-mobile/fenix/issues/new?template=release_checklist.md&title=Releng+for+) in the *upcoming* milestone: "Releng for v[release]". Find an engineer who will handle the next releng task and assign them.
+- [ ] Go through the list of issues closed during this sprint and make sure they all have the correct milestone.
+- [ ] Add `eng:qa:needed` flags on each issue that still needs it.
+- Note: You will need code review to make changes to the release branch after this point, because it is a protected branch.
+    - [ ] Push the branch.
 
-- [ ] Tag first pre-release RC version in Github. For 1.0 the tag would be 1.0.0-rc.1. This will kick off a build of the branch. You can see it in the mouseover of the CI badge of the branch in the commits view.
+- [ ] On GitHub, draft a GitHub Release with with a tag of the format `v2.3.0-rc.1` with the release branch as the target. Check the pre-release checkbox. This will kick off a build of the branch. You can see it in the mouseover of the CI badge of the branch in the commits view.
 
-    If you need to trigger a new RC build, you need to draft a publish a new (pre-release) release. Editing an existing release and creating a new tag will not trigger a new RC build.
+    If you need to trigger a new RC build, you need to draft and publish a new (pre-release) release. Editing an existing release and creating a new tag will not trigger a new RC build.
 
 - [ ] Upload the APK from the Taskcluster signing task to the [releases page](https://github.com/mozilla-mobile/fenix/releases).
 - [ ] Create a new PI request in Jira. You can clone [this issue](https://jira.mozilla.com/browse/PI-219).
 
+
+### SUMO Verification [Thursday, 2nd week of sprint]
+- [ ] If the *current* release is a major (x.0) release, review the SUMO article contents of the whats new / other sumo pages and make sure they are accurate with what is in this release. If not, escalate to Product Owner.
```

Comment:
For the most part, I think this would be looking at the list of features (which should be max 10) and seeing if they are completed. This **could** be complicated for long-running features (like ETP) but we should just be better about documenting so it's clear, I guess?